src/layerInfos/HandicapLayerInfo/HandicapLayerInfo.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+
+import './HandicapLayerInfo.scss';
+
+const propTypes = {
+  t: PropTypes.func.isRequired,
+  language: PropTypes.string.isRequired,
+  properties: PropTypes.object.isRequired,
+};
+
+const defaultProps = {};
+
+const names = {
+  barrierfree: 'Barrierefreie Bahnhöfe',
+  notBarrierfree: 'Nicht barrierefreie Bahnhöfe',
+  stuetzpunkt: 'bahnhof_plural',
+};
+
+const desc = (name, lng) => {
+  if (lng === 'de') {
+    return (
+      <div>
+        {name}, an denen beim SBB Call Center Handicap unter der Nummer{' '}
+        <a href="tel:0800 007 102">0800 007 102</a> eine Hilfestellung bestellt
+        werden kann.
+      </div>
+    );
+  }
+  if (lng === 'fr') {
+    return (
+      <div>
+        {name} pour lesquelles il est possible de commander un service
+        d’assistance auprès du Call Center Handicap CFF au numéro{' '}
+        <a href="tel:0800 007 102">0800 007 102</a>.
+      </div>
+    );
+  }
+  if (lng === 'en') {
+    return (
+      <div>
+        {name} where assistance can be requested from the SBB Call Center
+        Handicap by calling <a href="tel:0800 007 102">0800 007 102</a>.
+      </div>
+    );
+  }
+  return (
```

Comment:
Normally, de is the default language.

And why don't you use an object such as in https://github.com/geops/trafimage-maps/blob/master/src/layerInfos/PunctualityLayerInfo/PunctualityLayerInfo.js#L22?

Reply:
I don't see a solution better than another, but I implemented as you proposed for the sake of consistency :+1:  thanks!